numba/cuda/cuda_paths.py

```diff
@@ -77,21 +79,45 @@ def _cudalib_path():
         return 'lib64'
 
 
+def _static_cudalib_path():
+    if IS_WIN32:
+        return ('lib', 'x64')
+    else:
+        return ('lib64',)
+
+
 def _get_cudalib_dir_path_decision():
     options = [
         ('Conda environment', get_conda_ctk()),
+        ('Conda environment (NVIDIA package)', get_nvidia_cudalib_ctk()),
         ('CUDA_HOME', get_cuda_home(_cudalib_path())),
         ('System', get_system_ctk(_cudalib_path())),
     ]
     by, libdir = _find_valid_path(options)
     return by, libdir
 
 
+def _get_static_cudalib_dir_path_decision():
+    options = [
+        ('Conda environment', get_conda_ctk()),
+        ('Conda environment (NVIDIA package)', get_nvidia_static_cudalib_ctk()),
+        ('CUDA_HOME', get_cuda_home(*_static_cudalib_path())),
+        ('System', get_system_ctk(_cudalib_path())),
+    ]
+    by, libdir = _find_valid_path(options)
+    return by, libdir
```

Comment:
Temporaries needed? Could this just return `_find_valid_path(options)` ?

Reply:
It could, but then what's being returned is not evident from this function's code alone. I'd prefer to leave them in because it's already hard enough to follow all the path detection and library-finding logic, without making what's going from `_find_valid_path()` out the return value of this function hidden.

Reply:
Thanks for the explanation, am happy to leave as-is.